.github/workflows/release-publish.yml

```
@@ -0,0 +1,48 @@
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+
+      - name: Gradle publish
+        run: |
+          ./gradlew \
+          -PmavenCentralUsername=${{secrets.OSSRH_USERNAME}} \
+          -PmavenCentralPassword=${{secrets.OSSRH_PASSWORD}} \
+          -PsigningInMemoryKeyId=${{secrets.OSSRH_GPG_SECRET_KEY_ID}} \
+          -PsigningInMemoryPassword=${{secrets.OSSRH_GPG_SECRET_KEY_PASSWORD}} \
+          ${{ matrix.target }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)